README.md

```diff
@@ -69,12 +70,14 @@ yarn gulp <taskname> [[platform] [platform] ...]
 ```
 
 List of possible values of `<task-name>`:
-* **dist** copies all the JS and CSS files in the `./dist` folder.
+* **dist** copies all the JS and CSS files in the `./dist` folder [2].
 * **apps** builds the apps in the `./apps` folder [1].
-* **debug** builds debug version of the apps in the `./debug` folder [1].
+* **debug** builds debug version of the apps in the `./debug` folder [1][3].
 * **release** zips up the apps into individual archives in the `./release` folder [1]. 
 
 [1] Running this task on macOS or Linux requires Wine, since it's needed to set the icon for the Windows app (build for specific platform to avoid errors).
+[2] For Android platform, **dist** task will generate the `./cordova` folder
```

Comment:
I can rename the `cordova` folder to `cordova-dist`. But I think, there is a risk of confusion, the equivalent of the `dist` folder in the cordova project is more `cordova/src`. Moreover, the `cordova` folder can not be deleted like the `dist` folder

Reply:
Part of the problem is that the statement 'For Android platform, **dist** task will generate the `./cordova` folder' will make people think that this directory can be deleted when cleaning up after a build. Another problem is that `yarn gulp clean` currently fails to remove build artefacts in `/cordova` - the easiest solution for this will be to separate the files that currently end up under `/cordova` into two different directories: One for the permanent files, another one (which can be a sub-directory) for the build artefacts. This will also make fixing `yarn gulp clean` a lot easier.
But this can all be done in a follow-up pull request.

Reply:
@WalcoFPV: I think this needs to be fixed with some urgency, and it is a little bit more complex than previously thought:
Since a build of the Android app modifies some files that are under source control (`cordova/config.xml`, `cordova/config.xml`) these will appear as changed and sneak into subsequent commits for unrelated changes:

![image](https://user-images.githubusercontent.com/4742747/86523169-a172b600-bebc-11ea-9efb-717237d42a28.png)

This is a hassle and will require a lot of manual cleanup in the future, so it should be fixed - probably best rename the files to `..._template` or suchlike, and create a copy of them in the directory that is to contain the build artefacts.

Reply:
Done ! #2109